src/cargo/ops/cargo_test.rs

```diff
@@ -140,11 +140,19 @@ fn run_doc_tests(
     let mut errors = Vec::new();
     let config = options.compile_opts.config;
 
+    let mut runtool = None;
     // We don't build/run doc tests if `target` does not equal `host`.
     if compilation.host != compilation.target {
-        return Ok((Test::Doc, errors));
+        //currently does not support passing args along with the runtool
```

Comment:
Ugh I wish this undocumented behavior just didn't exist at all...

Reply:
What are you referring to? It does not support passing args because I just take the first parameter in the sequence, since I can't tell when args for the runtool end and other args for cargo begin

Reply:
I'm referring to the behavior (learned from the source code) that `runner = "..."` in the cargo configuration does not just specify a `PathBuf`, and that this is [not documented](https://doc.rust-lang.org/cargo/reference/config.html).

Reply:
I see